src/sbt-test/sbt-header/simple/build.sbt

```diff
@@ -0,0 +1,21 @@
+import de.heikoseeberger.sbtheader.license.Apache2_0
```

Comment:
You can just use any other setting or task key and call `value`: `scalaSource.in(Compile).value`